ui/app/src/components/shared/AssetList.vue

```diff
@@ -3,11 +3,11 @@
     <div class="line" v-for="item in items" :key="item.asset.symbol">
       <AssetItem class="token" :symbol="item.asset.symbol" />
       <div class="amount">
-        {{ formatNumber(item.amount) }}
+        {{ formatAssetAmount(item.amount) }}
```

Comment:
Is this intentional? We agreed not to use this function, right?

Reply:
This is so this branch is low touch. We are using the format function under the hood there.